src/java/org/apache/cassandra/utils/HeapUtils.java

```diff
@@ -130,7 +130,7 @@ public static String maybeCreateHeapDump()
                 }
                 else
                 {
-                    logger.debug("Heap dump creation on uncaught exceptions is disabled.");
+//                    logger.debug("Heap dump creation on uncaught exceptions is disabled.");
```

Comment:
will add back, way too much spam in the test logs making it hard to find *real* issues...